### PR TITLE
ninjas2: init at 0.2.0

### PR DESCRIPTION
**Pull request by magnetophon**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/ninjas2/default.nix [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  mkdir -p $out/lib/vst
+  cp bin/ninjas2-vst.so  $out/lib/vst
+  mkdir -p $out/bin
+  cp bin/ninjas2 $out/bin
+  '';
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/clearly-broken-software/ninjas2;
```
Comment (AndersonTorres) [inline pkgs/applications/audio/ninjas2/default.nix:36]:
```suggestion
    homepage = "https://github.com/clearly-broken-software/ninjas2";
```

Comment (magnetophon) [reply pkgs/applications/audio/ninjas2/default.nix]: Done, thanks!

File: pkgs/applications/audio/ninjas2/default.nix [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  patchPhase = ''
+    patchShebangs dpf/utils/generate-ttl.sh
+  '';
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [
+    libjack2 xorg.libX11 libGL mesa libsndfile libsamplerate
+  ];
+
+  installPhase = ''
```
Comment (AndersonTorres) [inline pkgs/applications/audio/ninjas2/default.nix:24]: Indent the lines below.

Comment (magnetophon) [reply pkgs/applications/audio/ninjas2/default.nix:24]: Thanks, done.

**Comment by magnetophon** [comment]
Fixed conflicts.

File: pkgs/applications/audio/ninjas2/default.nix [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    mkdir -p $out/lib/lv2
+    cp -r bin/ninjas2.lv2 $out/lib/lv2
```
Comment (Mic92) [inline pkgs/applications/audio/ninjas2/default.nix:26]: nitpick:

```suggestion
    install -D bin/ninjas2.lv2 $out/lib/lv2/ninjas2.lv2
```

Comment (Mic92) [reply pkgs/applications/audio/ninjas2/default.nix:26]: Same for the others.

Comment (magnetophon) [reply pkgs/applications/audio/ninjas2/default.nix]: Thanks, done.

**Comment by AndersonTorres** [comment]
ofBorg is failing; why?

**Comment by magnetophon** [comment]
@AndersonTorres I missed that, good catch!
Fixed now.

**Pull request merged**